src/post-py/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.6.0-alpine
+
+WORKDIR /app
+COPY . /app
```

Comment:
This code is changed frequently and it will reset your docker images cache so move it below installation of dependencies

Reply:
In this case, this option copies the file `requirements.txt` inside the image, pip install will not work without this file.

Reply:
copy requirements.txt separately and install the dependencies first (they are changed not so frequently as application code)